sklearn/model_selection/_split.py

```diff
@@ -376,7 +376,7 @@ class KFold(_BaseKFold):
         .. versionchanged:: 0.22
             ``n_splits`` default value changed from 3 to 5.
 
-    shuffle : boolean, optional
+    shuffle : bool, default=None
```

Comment:
the default is False

Reply:
Thanks for the feedback @NicolasHug ! I've corrected my work and pushed changes.